docs: outdated description of Go version in CONTRIBUTING.md

Run `make build` now gives the following error message after updating to https://github.com/cockroachdb/cockroach/commit/ce9834efc4b62ca4a77389dbbb7e746068f6ae61:

```
go1\.7.* required (see CONTRIBUTING.md): go version go1.6.2 linux/amd64
make: *** [.go-version] Error 1
```

But [CONTRIBUTING.md](https://github.com/cockroachdb/cockroach/blob/master/CONTRIBUTING.md) still says:

> A Go environment with a 64-bit version of Go 1.6.2.

In the Git commit log, I find:

>    Bump to go1.7rc3

After I upgrade to  go1.7rc3, the error message is gone.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -5,7 +5,7 @@
 1.  Install the following prerequisites, as necessary:
   - A C++ compiler that supports C++11. Note that GCC prior to 6.0 doesn't
   work due to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891.
-  - A Go environment with a 64-bit version of Go 1.6.2.
+  - A Go environment with a 64-bit version of Go 1.7.
   - Git 1.8+
   - Bash
 

--- a/acceptance/cluster/localcluster.go
+++ b/acceptance/cluster/localcluster.go
@@ -54,7 +54,7 @@ import (
 
 const (
 	builderImage     = "cockroachdb/builder"
-	builderTag       = "20160808-065131"
+	builderTag       = "20160816-112252"
 	builderImageFull = builderImage + ":" + builderTag
 	networkName      = "cockroachdb_acceptance"
 )

--- a/build/Dockerfile
+++ b/build/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.7rc5
+FROM golang:1.7
 
 MAINTAINER Peter Mattis <peter@cockroachlabs.com>
 

--- a/build/build-osx.sh
+++ b/build/build-osx.sh
@@ -5,4 +5,4 @@ set -eu
 go get -u github.com/karalabe/xgo
 # OSX 10.9 is the most recent version available at time of writing.
 # If changing the OS/arch target, adjust the filename in push-aws.sh.
-xgo --image cockroachdb/xgo:20160808 --targets=darwin-10.9/amd64 --go=1.7rc5 --ldflags="$($(dirname "${0}")/ldflags.sh)" ${GOPATH%%:*}/src/github.com/cockroachdb/cockroach
+xgo --image cockroachdb/xgo:20160816 --targets=darwin-10.9/amd64 --go=1.7 --ldflags="$($(dirname "${0}")/ldflags.sh)" ${GOPATH%%:*}/src/github.com/cockroachdb/cockroach

--- a/build/xgo/Dockerfile
+++ b/build/xgo/Dockerfile
@@ -1,4 +1,4 @@
-FROM karalabe/xgo-1.7rc5
+FROM karalabe/xgo-1.7
 
 MAINTAINER Tamir Duberstein <tamird@gmail.com>
 

